setup.md

```diff
@@ -176,16 +176,11 @@ iOS devices.
    $ brew update
    $ brew install --HEAD libimobiledevice
    $ brew install ideviceinstaller ios-deploy
+   $ brew install cocoapods
+   $ brew link cocoapods
```

Comment:
Yep; just `brew install` is sufficient & consistent with our other homebrew instructions.